docker-compose.override.yml

```diff
@@ -25,6 +26,7 @@ services:
     fpm:
         volumes:
             - ./:/srv/journal
+            - ./.docker/php.ini:/usr/local/etc/php/conf.d/x-dev.ini
```

Comment:
Usual convention is to use numbers eg (`99-`), but none of the other files follow it so would still come first.